test/inductor/test_torchinductor_strided_blocks.py

```diff
@@ -0,0 +1,261 @@
+# Owner(s): ["module: inductor"]
+import contextlib
+import functools
+import importlib
+import unittest
+from typing import Any, Callable, Iterable, Optional, Tuple
+
+import torch
+import torch.utils._pytree as pytree
+from torch._inductor import config
+from torch._inductor.runtime.hints import TRITON_MAX_BLOCK
+from torch._inductor.runtime.runtime_utils import is_power_of_2
+from torch._inductor.test_case import TestCase as InductorTestCase
+from torch._inductor.utils import run_and_get_code
+from torch.testing._internal.common_utils import (
+    instantiate_parametrized_tests,
+    parametrize,
+)
+from torch.testing._internal.inductor_utils import GPU_TYPE, HAS_GPU, skip_windows_ci
+
+
+requires_gpu = functools.partial(unittest.skipIf, not HAS_GPU, "requires gpu")
+
+skip_windows_ci(__name__, __file__)
```

Comment:
I refactored this blob into `inductor_utils` so we wouldn't have to copy+paste it in every test.